### ISSUE TITLE
Playback starts when I don't want it to

### ISSUE THREAD

**Issue by avh4** [navidrome/navidrome#1845]
### Description

Navidrome randomly starts playing when I don't want it to.

### Expected Behaviour

Playback should not start unless I click the "Play" button.

### Steps to reproduce

Restarts when waking from sleep:
1. Add multiple songs to the play queue and start playing
2. Pause playback
3. Put the computer to sleep
4. Wake the computer
5. Playback starts while the OS is still showing the lock screen / password entry

Restarts when reopening the webapp:
1. Add multiple songs to the play queue and start playing
2. Pause playback
3. Close the browser tab
4. Open navidrome again in a new tab
5. Perform some navigation action (either clicking to another page, or typing in the search field)
6. Playback starts

### Platform information

 - Navidrome version: [0.47.5](https://github.com/navidrome/navidrome/releases/tag/v0.47.5) (86fe1e3b)
 - Browser and version: Firefox 103.0.2 (64-bit) Snap for Ubuntu canonical-002 - 1.0
 - Operating System: Client is Ubuntu 22.04.1 LTS.  Server is running in Docker version 20.10.17-qnap1, build 0474f29.

### Additional information

Any other information that may be relevant or give context to the problem.

 - Logs? (not sure where to find these)
 - Client used: Web client

**Comment by vs49688** [comment]
This is probably something screwy with MPRIS and your screen locker/DE trying to be nice. I have the same behaviour on KDE when I have `Pause media players when suspending` enabled. I suspect Navidrome/Firefox isn't tracking the play/pause state correctly.

**Comment by avh4** [comment]
Oh, I am using the KDE desktop with Ubuntu.  I'll try turning off that KDE setting and see if it solves it.  For future reference, I found the setting in KDE System Settings -> Power Management -> Advanced Power Settings -> Other Settings -> Pause Media players when suspending.

**Comment by avh4** [comment]
Turning off that KDE setting does fix the issue when waking the computer from sleep, but does not fix the issue when opening a new browser window after closing the previous one.  I guess I should split this into separate issues then.
